src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -1,19 +1,35 @@
 import React, { Dispatch, FC, MouseEvent, SetStateAction, useEffect, useMemo } from 'react';
+import { Trans } from 'react-i18next';
```

Comment:
please break this file to small pieces, its getting too big

Reply:
Moved the popover icon outside